app/src/main/java/org/jellyfin/androidtv/details/trailers/YouTubeTrailerPresenter.kt

```diff
@@ -0,0 +1,78 @@
+package org.jellyfin.androidtv.details.trailers
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.util.Log
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.Toast
+import androidx.appcompat.view.ContextThemeWrapper
+import androidx.leanback.widget.Presenter
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.resource.drawable.GlideDrawable
+import com.bumptech.glide.request.RequestListener
+import com.bumptech.glide.request.target.Target
+import kotlinx.android.synthetic.main.multi_badge_image_card_view.view.*
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.base.IItemClickListener
+import org.jellyfin.androidtv.model.trailers.external.YouTubeTrailer
+import org.jellyfin.androidtv.ui.MultiBadgeImageCardView
+import org.jellyfin.androidtv.util.ImageUtils
+
+class YouTubeTrailerPresenter(private val context: Context, private val imageHeight: Int, private val safeBrandingCompliance: Boolean) : Presenter(), IItemClickListener {
```

Comment:
When a class (or function for that matter) gets multiple parameters the code might look a bit messy. A way to prevent this is by putting each parameter on it's own line:
```kotlin
class YouTubeTrailerPresenter(
	private val context: Context,
	private val imageHeight: Int,
	private val safeBrandingCompliance: Boolean
) : Presenter(), IItemClickListener {
```